### PR TITLE
#167278120setup database

### PR DESCRIPTION
**Pull request by Kitingu**
**what does this PR do?**
set up a database for persisting data
**Description of Task to be completed**
`add drop tables file`
`add create tables file`
`add database querying file`

**How should this be manually tested?**
`Clone this repo`
`cd into the main directory`
`run npm install to install required dependencies`
`Set the required environment variables by editing the sample_env file`
`git checkout ch-setup-database-167278120`
`run npm createTables to create tables `
`run npm  drop to drop the tables`

**What are the relevant pivotal tracker stories**
https://www.pivotaltracker.com/story/show/167278120

**Screenshots (if appropriate)**
Screenshot from 2019-07-16 02-54-57

### CODE REVIEW

File: server/models/db/db.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const config = require('../../../config');
+const DATABASE_URL = config.appConfig.DATABASE_URL
+const { Pool } = require('pg');
```
Comment (njeri-ngigi) [inline server/models/db/db.js:3]: update your package.json file to include the 'pg' module

File: server/models/db/drop-tables.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const { Pool } = require('pg');
+
+
+const config = require('../../../config')
+
+const DATABASE_URL = config.appConfig.DATABASE_URL
```
Comment (njeri-ngigi) [inline server/models/db/drop-tables.js:6]: this URL is missing in your config.js file.

File: server/models/db/db.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+const config = require('../../../config');
```
Comment (njeri-ngigi) [inline server/models/db/db.js:1]: In your config.js you can get the database URL from the .env file for development environment

**Review by njeri-ngigi — COMMENTED** [review]
Your instructions on how to test this PR are a bit unclear. You need to add a step on how to create the databases, and where to add the DATABASE_URL

File: server/models/db/db.js (hunk 3 above)
Comment (Kitingu) [reply server/models/db/db.js:1]: I have updated the files now everything works fine.

File: server/models/db/drop-tables.js (hunk 2 above)
Comment (Kitingu) [reply server/models/db/drop-tables.js:6]: Covered.

**Pull request merged**